lib/fastlane/setup/setup_ios.rb

```diff
@@ -72,6 +79,29 @@ def generate_appfile
       Helper.log.info "Created new file '#{path}'. Edit it to manage your preferred app metadata information.".green
     end
 
+    def default_run_produce
+      Helper.log.info "Running produce..."
+      require 'produce'
+      config = {}
+      FastlaneCore::Project.detect_projects(config)
+      project = FastlaneCore::Project.new(config)
+
+      produce_options_hash = {
+          app_name: project.default_app_name
+      }
+      Produce.config = FastlaneCore::Configuration.create(Produce::Options.available_options, produce_options_hash)
+      begin
+        ENV['PRODUCE_APPLE_ID'] = Produce::Manager.start_producing
+      rescue => exception
+        if exception.to_s.include?("The App Name you entered has already been used")
+          Helper.log.info 'It looks like that App Name has already been taken, please enter an alternative.'.yellow
```

Comment:
`enter an alternative`, the user never entered one, so this message should be more clear, something like
> It looks like that App Name of your project has already been taken on iTunes Connect or the Apple Developer Portal. Please provide a name for your app: